executor/partition_table_test.go

```diff
@@ -953,10 +954,9 @@ func TestGlobalStatsAndSQLBinding(t *testing.T) {
 	tk.MustExec("analyze table trange")
 	tk.MustExec("analyze table tlist")
 
-	// after analyzing, the planner will use the Index(a)
-	tk.MustIndexLookup("select * from thash where a<100")
-	tk.MustIndexLookup("select * from trange where a<100")
-	tk.MustIndexLookup("select * from tlist where a<1")
+	require.True(t, tk.HasPlan("select * from thash where a<100", "TableFullScan"))
```

Comment:
Expected, model2 prefers to use Scan instead of Lookup to avoid triggering too many double-read requests.